src/ui/components/lobbyList/lobbyListComponent.ts

```diff
@@ -6,34 +6,57 @@ const Vue = <vuejs.VueStatic>require("vue");
 
 import "../../css/lobby-list.less";
 import NetworkService from "../../services/networkService.ts";
+import StaticService from "../../services/staticService.ts";
 
 @Component({
     template: require("./lobbyListView.html")
 })
 export default class LobbyListComponent extends Vue {
-    private username: string;
+    private lobbySelected : lobby.LobbyListItem;
+    private canCreateLobby : boolean;
     
     created() {
+        this.canCreateLobby = true;
         // Mainly for debugging. Redirects if theres no connection.
         if (!NetworkService.currentConnection) {
-            this.$router.go("/loading");
+            this.$router.go("/login");
         }
     }
     
     data() {
         return {
-            username: "Player",
-            network: NetworkService
+            network: NetworkService,
+            staticService: StaticService,
+            lobbySelected: undefined,
+            canCreateLobby: this.canCreateLobby
```

Comment:
See my previous comment about `data`. You should also prefer `null` over `undefined`.